src/status_im/chat/models/mentions.cljs

```diff
@@ -12,6 +12,157 @@
 
 (def at-sign "@")
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
why this part is separated? just wondering

Reply:
Because file is ишп фдкуфвн and needed some visual separation фы i was tired by this code :D I thought about moving it to a separate ns but I didn't find a proper name for this piece of... code and just left it here.